src/main.rs

```diff
@@ -32,65 +32,65 @@ const CARGO_PKG_NAME: &str = env!("CARGO_PKG_NAME");
 const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 
 #[derive(Debug, Parser)]
-#[clap(name = format!("{CARGO_PKG_NAME} (v{CARGO_PKG_VERSION})"))]
+#[clap(name = CARGO_PKG_NAME)]
```

Comment:
If you remove this line, clap will automatically fill in `name = CARGO_PKG_NAME` for you